frontend/src/components/Tag.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Tag as TrussTag } from '@trussworks/react-uswds';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import classnames from 'classnames';
+import './Tag.scss';
+
+export default function Tag({
+  children, className, clickable, handleClick,
+}) {
+  const tagClass = classnames('ttahub-tag', className, {
+    'ttahub-tag-underline': clickable,
+  });
+  return (
+    <TrussTag className={tagClass} onClick={handleClick}>
```

Comment:
In the past we have tried to use the defined USWDS styles in place of raw CSS as it keeps our values in sync. You might be able to change a few of these to use the USWDS values,

See this for ref: https://designsystem.digital.gov/utilities/display/

Reply:
Updated, thanks!